examples/ticket-sales/index.mjs

```diff
@@ -0,0 +1,38 @@
+import { loadStdlib } from "@reach-sh/stdlib";
+import * as backend from './build/index.main.mjs';
+const stdlib = loadStdlib({REACH_NO_WARN: 'Y'});
+const MAX = 50;
+const accA = await stdlib.newTestAccount(stdlib.parseCurrency(5000));
+const ctcA = accA.contract(backend);
+const tickets = await stdlib.launchToken(accA, "Tickets", "TIX", {supply: MAX});
+
+console.log('Welcome to the ticket distributor\nLets get you a ticket');
+
+const startBuyers = async () => {
+  await stdlib.wait(10);// need this to work on ETH
+  const runBuyers = async (i) => {
+    const acc = await stdlib.newTestAccount(stdlib.parseCurrency(100));
+    const ctc = acc.contract(backend, ctcA.getInfo());
+    await acc.tokenAccept(tickets.id);
+    await ctc.apis.Buyer.buyTicket();
+    console.log(`Tickets sold: ${i}`);
+  }// end of runBuyers
+  for(let i = 0; i < MAX; i++){
+    await runBuyers(i);
```

Comment:
```suggestion
    await runBuyer(i);
```

This runs them one at a time. If you want them in parallel, then you'd use `Promise.all`